cartographer/mapping/3d/submap_3d.cc

```diff
@@ -261,7 +261,7 @@ void Submap3D::ToResponseProto(
 
 void Submap3D::InsertRangeData(const sensor::RangeData& range_data,
                                const RangeDataInserter3D& range_data_inserter,
-                               const int high_resolution_max_range) {
+                               const double high_resolution_max_range) {
```

Comment:
Let's change these to `float`. It is quite common that options are stored as `double`, since there is no need for float support in Lua options. The data this gets compared against is `float`, so keeping it as `double` does not provide any benefit.